File: program76.c
// Input :  4
// Output :        -4   -3  -2  -1  0   1   2   3   4

#include<stdio.h>

void Display(int iNo)   // 3
{
       int iCnt =0;
        //          1               2               3
       for(iCnt = -iNo; iCnt <= iNo; iCnt++)
       {
            printf("%d\t",iCnt);        // 4
       }
}

int main()
{
    int iValue =0;

    printf("Please enter the value :\n");
    scanf("%d",&iValue);

    Display(iValue);    // Display(3)

    return 0;
}

File: program76.java
import java.lang.*;
import java.util.*;

class Pattern
{
	private int iNo;
	private char ch;
			
	public void Accept()
	{
		Scanner sobj = new Scanner(System.in);
		
		System.out.print("Enter the number : ");
		iNo = sobj.nextInt();
	}
	public void Display()
	{
		int iCnt;
		if(iNo > 0)
		{
			for(iCnt = iNo ; iCnt >= 1 ; iCnt--)
			{
				System.out.print(iCnt+" # ");
			}
		}
		else
		{
			System.out.println("Enter positive number");
		}
	}
}
class program76
{
	public static void main(String args[])
	{
		Pattern obj = new Pattern();
		obj.Accept();
		obj.Display();
	}
}